mergify_engine/engine/actions_runner.py

```diff
@@ -211,11 +211,10 @@ async def get_summary_check_result(
 ) -> typing.Optional[check_api.Result]:
     summary_title, summary = await gen_summary(ctxt, pull_request_rules, match)
 
-    serialized_conclusions = serialize_conclusions(conclusions)
-    summary_for_logging = summary + serialized_conclusions
+    summary += serialize_conclusions(conclusions) + summary
```

Comment:
```suggestion
    summary = serialize_conclusions(conclusions) + summary
```
like this right ? otherwise we have summary + serialize_conclusions(conclusions) + summary ?